### ISSUE TITLE
About training the Food101N data

### ISSUE THREAD

**Issue by Camelliahmy** [kuanghuei/clean-net#5]
Do you directly fine-tune the model pre-trained on ImageNet on the noisy data in Food101N without using any verification labels? I trained the ResNet50 model on the noisy data but only get 77% top-1 accuracy (81.44% in your paper). I was wondering if you could share the training code or hyperparameters of the baseline model? Thank you very much :)

**Comment by DayChan** [comment]
Have you figured out how to approach 81%?

**Comment by House-Leo** [comment]
Would you mind provide the .tsv file you trained to me?
